### ISSUE TITLE
Add notes to editable files about NOT using windows "wordpad"

### ISSUE THREAD

**Issue by ntoff** [guysoft/OctoPi#441]
Twice now I've seen an issue on the G+ community where someone has used windows' "Wordpad" to edit octopi.txt and it's broken it. Perhaps the same warning that applies to Text Edit's evil EVIL "smart quotes" needs to be extended to include not using Wordpad *at all*, and also a warning note put into the octopi.txt file about only using recommended text editors like sublime or notepad++ if you're on windows.

**Comment by guysoft** [comment]
We have [this warning](https://github.com/guysoft/CustomPiOS/blob/CustomPiOS/src/modules/network/filesystem/boot/custompios-network.txt#L13), If you have an idea how to extend this message you are welcome to pull request it.
It has instructions how to not get around this issue for mac users. I don't recommend people download something new. I am pretty sure windows have a way to edit plaintext files.

**Comment by ntoff** [comment]
That warning doesn't exist in octopi.txt (no warning does), and warnings for mac users don't help anyone using windows, and using a different piece of software.

Windows wordpad doesn't just break the quotes, it also breaks the line endings, it totally screws up the file in more ways than textedit does. 

Windows uses a completely different line ending format https://en.wikipedia.org/wiki/Newline so users need to edit the files using a file editor that is aware of unix style line endings.

https://plus.google.com/+morherman/posts/RnevFz8QHKC

This is the second post I've seen where someone has used wordpad to edit octopi.txt, completely breaking it. Notepad++ is a trusted and well known text editor (and free) so if you don't suggest they download anything new, a warning should still be in place that using wordpad will completely obliterate the config.

**Comment by guysoft** [comment]
I said there is "this warning", and if you want to PR one for windows users you are welcome.

**Comment by nophead** [comment]
How about a startup script that detects bad line endings and quotes and
simply fixes them? Can't be that difficult and much better than warnings
that get ignored.

**Comment by guysoft** [comment]
I'd accept a PR like that too

**Comment by ntoff** [comment]
warnings in the text file I can handle, I don't know how to do the format checking script though.

**Comment by ntoff** [comment]
I started on the extra warnings but as nophead said, they'd probably get ignored anyway, and the configureation files are cluttered with comments enough as it is. 

Either centralised documentation with details on editing (like a `README BEFORE EDITING` file in /boot), or just the checking script would probably be neater. Unfortunately users tend not to read documentation, and I don't know how to do the checking script.

**Comment by guysoft** [comment]
Indeed, so should I close this? Or is there some action that can be done?

**Comment by ntoff** [comment]
It's up to you, you can leave it open if you want, as a reminder something might need doing some day, like some sort of checking script, or just close it if you don't plan on doing anything about it (like I said I don't know how to do the script, otherwise I'd do a PR).

**Comment by John-Mc** [comment]
It would also be helpful if the Getting Started With OctoPi instructions at https://octoprint.org/download/ made some mention of the fact that there are issues with the two programs that people new to this are probably most likely to use (Windows Wordpad, Apple's TextEdit. Not sure if Windows Notepad also has problems?)

The only mention I can find of Notepad problems is a brief mention at about 4:25 in Tom's "Getting started..." video on that download page. That is so quick it's easy to miss, and a lot of people will work from the text description, rather than the video.

**Comment by guysoft** [comment]
We always get the feeling people never read, but @foosel , you can probably add that to the page. Might get a few people in the right direction.

**Comment by foosel** [comment]
[x] done

**Comment by John-Mc** [comment]
Cool!  I use BBEdit on the Mac. The free version has all the features I need (all of the features of the now-discontinued TextWrangler are in the free version of BBEdit)
